vueTraining/vuetask-3/src/components/AddTodo.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div id="add-todo">
+    <h2>タスクの追加</h2>
+    <input
+      type="text"
+      name="input-add-task"
+      id="input-add-task"
+      v-model="task"
+    />
+    <!-- <input type="button" @click="addTask" value="追加" /> -->
+    <button @click="addTask">追加</button>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'AddToDo',
+  data: () => {
+    return {
+      task: '',
+    };
+  },
+  methods: {
+    addTask: function () {
+      const id = this.$store.getters.getId;
+      const todo = { id: id, task: this.task, status: '作業中' };
```

Comment:
細かい点ですが、オブジェクトはkeyとvalueの文字が一致していれば以下のように省略が可能です！
```suggestion
      const todo = { id, task: this.task, status: '作業中' };
```
現場ではよく使用される省略記法ですので、覚えておくと良いです！

Reply:
ありがとうございます！意識して省略するようにしたいと思います。